olp-cpp-sdk-dataservice-read/include/olp/dataservice/read/CatalogVersionRequest.h

```diff
@@ -113,6 +109,26 @@ class DATASERVICE_READ_API CatalogVersionRequest final {
     fetch_option_ = fetchoption;
     return *this;
   }
+
+  /**
+   * @brief Creates readable format for the request.
+   * @param none
+   * @return string representation of the request
+   */
+  std::string CreateKey() const {
```

Comment:
mark inline

Reply:
Compilers may decide to (un)inline the method regardless of the keyword.
I don't see the point of making it inline forcibly either.

Reply:
https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#Rs-inline
https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#Rf-inline
Not always the case, but for in-class defines this is true. Either mark all `inline` or none. Do not leave half states.

Reply:
Makes sense, I updated the definition.